squirrel_8dof_planner/launch/planner.launch

```diff
@@ -10,7 +10,8 @@
   <group ns="$(arg planning_ns)">
     <rosparam command="load" file="$(find squirrel_8dof_planner)/config/parameters.yaml" />
     <rosparam command="load" file="$(find squirrel_8dof_planner)/config/folding_poses_$(arg robot).yaml" />
-    <param name="robot_description" textfile="$(find squirrel_8dof_planner)/config/robotino_plan.urdf" />
+    <param name="robot_description_addition" textfile="$(find squirrel_8dof_planner)/config/robotino_description_addition"/>
```

Comment:
@philkark are you sure that this is correct?

Reply:
@ipa-nhg  Yes this should be correct. I load the global robot description in code, parse it, adapt it to the planner and then set the description parameter in code.